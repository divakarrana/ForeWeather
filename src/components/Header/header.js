import React from 'react';
import styles from './header.module.css';

const header = () => {
    return (
        <div className={styles.Header}>
            <div className={styles.logoBox}><i className="fas fa-smog"></i></div> 
            <h1 className={styles.appHeading}>Fore-Weather &nbsp;<span className={styles.subHeading}>5 day weather forecast for Mumbai</span></h1>
        </div>
    )
}

export default header;
