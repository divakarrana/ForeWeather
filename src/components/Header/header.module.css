.Header {
  display: flex;
  align-items: center;
  background-color: #111c3a;
  height: 8rem;
  box-shadow: var(--shadow-dark);
  color: var(--color-grey-light-1);
}

.logoBox {
  font-size: 4.5rem;
  padding: 0rem 2rem;
}

.appHeading {
  font-size: 2.5rem;
  font-weight: 300;
}

.subHeading {
  font-size: 1rem;
  font-weight: 600;
}
