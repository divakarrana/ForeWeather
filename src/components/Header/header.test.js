import React from 'react';
import ReactDOM from 'react-dom';
import {cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import Header from './header';

describe('<Header />', () => {

  afterEach(cleanup);

  it("should render without crashing <Header />", () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header></Header>, div)
  });


  it("should match snapshot <Header />", () => {
    const tree = renderer.create(<Header></Header>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});