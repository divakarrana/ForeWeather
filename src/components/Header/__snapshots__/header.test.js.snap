// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapshot <Header /> 1`] = `
<div
  className="Header"
>
  <div
    className="logoBox"
  >
    <i
      className="fas fa-smog"
    />
  </div>
  <h1
    className="appHeading"
  >
    Fore-Weather Â 
    <span
      className="subHeading"
    >
      5 day weather forecast for Mumbai
    </span>
  </h1>
</div>
`;
